client/api/MovieApi.js

```diff
@@ -8,4 +8,15 @@ export function getTopMovies() {
     let promise = apiClient.get('/getTopMovies/');
     const dataPromise = promise.then((response) => response.data);
     return dataPromise;
+}
+
+export function searchMovies(query) {
+    // headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+    let data = {"query":query};
+    console.log("data", data);
+    let promise = apiClient.post('/searchMovies/', 
```

Comment:
agree about the GET here ya, side bonus: allows the user to edit query params thru the URL (or you to modify query params via the URL, etc) - fun interface